package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "airnode",
+  "version": "0.1.0",
```

Comment:
We are still working to have the 0.1.0 version, so do you mind to change to 0.0.1?

Reply:
Totally! I'll modify right now